Promact.Oauth.Server/src/Promact.Oauth.Server/wwwroot/app/users/user-add/user-add.component.ts

```diff
@@ -71,31 +72,34 @@ export class UserAddComponent {
     }
 
     checkEmail(email) {
-        this.isEmailExist = false;
-
-        this.userService.findUserByEmail(email + "@promactinfo.com").subscribe((isEmailExist) => {
-            if (isEmailExist) {
-                this.isEmailExist = true;
-            }
-            else {
-                this.isEmailExist = false;
+        if (email !== "" && email !== undefined) {
+            this.isEmailExist = false;
+            this.userService.findUserByEmail(email + "@promactinfo.com").subscribe((isEmailExist) => {
+                if (isEmailExist) {
+                    this.isEmailExist = true;
+                }
+                else {
+                    this.isEmailExist = false;
 
-            }
-        }, err => {
-        });
+                }
+            }, err => {
+            });
+        }
     }
 
     checkSlackUserName(slackUserName) {
-        this.isSlackUserNameExist = false;
-        this.userService.findUserBySlackUserName(slackUserName).subscribe((isSlackUserNameExist) => {
-            if (isSlackUserNameExist) {
-                this.isSlackUserNameExist = true;
-            }
-            else {
-                this.isSlackUserNameExist = false;
-            }
-        }, err => {
-        });
+        if (slackUserName !== "" && slackUserName !== undefined) {
+            this.isSlackUserNameExist = false;
+            this.userService.findUserBySlackUserName(slackUserName).subscribe((isSlackUserNameExist) => {
+                if (isSlackUserNameExist) {
```

Comment:
Actually I will update only condition not added any variable, so i wll update this